scripts/jenkins/ardana/ansible/roles/heat_stack/tasks/main.yml

```diff
@@ -23,6 +23,15 @@
   register: get_stack
   failed_when: false
 
+  # backwards-compatibility: handle old stack naming
```

Comment:
Maybe you could add a note that we can remove this after some time, i.e. once we don't have running stacks with the old naming scheme anymore.

Reply:
I did you one better: I already created a PR for this that we will merge in one week or so: #3490

Reply:
Perfect! :+1: